docs/external_cmps.md

```diff
@@ -0,0 +1,81 @@
+# External components
+
+By default, plain ```docker-compose up```, regardless of the
+pipeline will start containers for following services:
+
+* MySQL
+* Keystone
+* ElasticSearch
+* Zookeeper
+* Kafka
+
+However each of them is treated, in the same time, as the
+dependency for *Monasca* hence can be externalized with the help
+of appropriate *environmental variables*.
+
+Following guide presents how to achieve that.
+
+## Using external OpenStack Keystone
+
+By default `monasca-docker` is using Keystone stored in separate
+container. But if you want to use external Keystone you need to
+overwrite some environment variables in specific containers.
+
+Remove `keystone` image initialization and all references to this
+image from `depends_on` from other images from
```

Comment:
..._on\` **,** from other images **,** from

Reply:
Done, additionally replaced `from other images` with `from other services`.

In compose files we have **services**, not **images**.

@matrixik's fault :dancer: